packages/cli/src/commands/destroy/scaffold/__tests__/scaffold.test.js

```diff
@@ -10,13 +10,7 @@ import { files } from '../../../generate/scaffold/scaffold'
 import { tasks } from '../scaffold'
 
 jest.mock('fs')
-jest.mock('@babel/core', () => {
```

Comment:
@peterp - do you remember why this was added? Was it to just make the tests pass?